internal/services/containers/kubernetes_cluster_other_resource_test.go

```diff
@@ -928,7 +952,7 @@ func TestAccKubernetesCluster_webAppRoutingPrivateDNS(t *testing.T) {
 				check.That(data.ResourceName).Key("web_app_routing.0.web_app_routing_identity.#").HasValue("1"),
 			),
 		},
-		data.ImportStep(),
+		data.ImportStep("web_app_routing.0.dns_zone_id", "web_app_routing.0.dns_zone_ids.#", "web_app_routing.0.dns_zone_ids.0"),
```

Comment:
Because in this test config, this feature is specified with `dns_zone_id`. But in the import step, by default it will be imported as `dns_zone_ids`, so there're some differences between them.

Reply:
This test will fail in 4.0 mode, so can we add a `t.Skip` for it?

Reply:
@ms-henglu, this test as well as `TestAccKubernetesCluster_webAppRouting` will fail in 4.0 mode, can you please add the skip for these.